quetz/tasks/mirror.py

```diff
@@ -106,6 +114,32 @@ def json(self):
         return json.load(self.file)
 
 
+def download_repodata(repository: RemoteRepository, channel: str, platform: str):
```

Comment:
Since this function is basically a wrapper around `fetch_repo_data`, I'd keep this one as an asynchronous function and do the `asyncio.run(...)` part in the caller for this function.

Reply:
Changed. I used `aiofiles` to make the files operation async then.

Reply:
Awesome!